dev/com.ibm.ws.java11_fat/src-reference/java16/org/gjwatts/liberty/TestService.java

```diff
@@ -0,0 +1,64 @@
+package org.gjwatts.liberty;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Objects;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@Path("/")
+@ApplicationScoped
+public class TestService {
+
+    @GET
+    public String test() {
+        try {
+            log(">>> ENTER");
+            doTest();
+            log("<<< EXIT SUCCESSFUL");
+        } catch (Exception e) {
+            e.printStackTrace(System.out);
+            e.printStackTrace(new PrintWriter(sb));
+            log("<<< EXIT FAILED");
```

Comment:
You are likely going down this path.  You are printing the stack trace of the exception to system.out and to the response header of the endpoint.
I would take a look at your server logs for the exception being thrown here and debug that.